pkgs/development/python-modules/thumbor/default.nix

```diff
@@ -27,18 +53,26 @@ buildPythonPackage rec {
     pillow
     derpconf
     python_magic
-    pexif
     libthumbor
-    opencv
+    opencv3
     webcolors
     piexif
     statsd
+    pkgs.exiftool
+    pkgs.libjpeg
+    pkgs.ffmpeg
+    pkgs.gifsicle
```

Comment:
AFAIK it's not needed to include `pkgs` into this expression, adding `exiftool` etc to the argument list should be sufficient

Reply:
Ok! I thought it'd be more explicit which packages are Python packages and which are system packages. Because there are name collisions at least with `redis`. Or is there some other way to request for the system package `redis` instead of Python package `redis` as an argument? As I needed to use `pkgs` for that, I thought why not use the same for all system packages. But I can remove `pkgs.` from all other except `pkgs.redis` if that's better.

Reply:
I guess you're right. Some packages have Python bindings with the same name in the Python package set. Adding `pkgs` makes it clearer :+1: